astropy/table/jsviewer.py

```diff
@@ -96,9 +96,23 @@ class Conf(_config.ConfigNamespace):
 
 
 class JSViewer(object):
-    def __init__(self,
-                 use_local_files=False,
-                 display_length=50):
+    """A class that provides an interactive HTML export of a Table.
```

Comment:
I'd drop the "A class that"--it's redundant and sort of beside the point that it happens to be a class.